src/mame/drivers/at.cpp

```diff
@@ -1286,6 +1306,20 @@ ROM_START( comportiii )
 	ROMX_LOAD( "109737-002.bin", 0x10001, 0x8000, CRC(8463cc41) SHA1(cb9801591e4a2cd13bbcc40739c9e675ba84c079), ROM_SKIP(1) | ROM_BIOS(2) )
 ROM_END
 
+// Ericsson WS286
+ROM_START(ews286 ) // Computer is brown/yellow-ish with Ericsson logo
+	ROM_REGION(0x20000,"bios", 0)
+	ROM_LOAD16_BYTE( "RYS_103_1002_R8A_3C00_IC-POS_71.BIN", 0x18000, 0x4000, CRC(af179e56) SHA1(58b1df46d6e68eef472a0529cb9317abaf17880f)) // Last ROM set and has Nokia
+	ROM_LOAD16_BYTE( "RYS_103_1003_R8A_8600_IC-POS_69.BIN", 0x18001, 0x4000, CRC(555502cb) SHA1(1977fe54b69c5e52731bf3eb8bdabe777aac014b)) // copyright patched in both roms
+ROM_END
+
+// Nokia Data WS286
+ROM_START(nws286 ) // Computer is grey with Nokia logo.
+	ROM_REGION(0x20000,"bios", 0)
+	ROM_LOAD16_BYTE( "RYS_103_1002_R8A_3C00_IC-POS_71.BIN", 0x18000, 0x4000, NO_DUMP)
+	ROM_LOAD16_BYTE( "RYS_103_1003_R8A_8600_IC-POS_69.BIN", 0x18001, 0x4000, NO_DUMP)
+ROM_END
```

Comment:
If there are no dumps at all for this machine then it should not be added as a driver. Add the information in comments instead.

Reply:
Ok, I am looking for a newer rom, but I can't tell when it pops up, I would think the ews286 works but I want to get it from an actual Nokia branded machine to be authentic, I'll comment it out for now.

Reply:
And I tried to use ews286 as parent but that didn't validate